x-pack/platform/plugins/shared/cases/server/features/v2.ts

```diff
@@ -62,7 +75,21 @@ export const getV2 = (): KibanaFeatureConfig => {
           all: [...filesSavedObjectTypes],
           read: [...filesSavedObjectTypes],
         },
-        ui: capabilities.all,
+        ui: [
+          ...capabilities.all,
+          ...capabilities.createComment,
+          ...capabilities.reopenCase,
```

Comment:
The ui privs should match above; remove `createComment` and `reopenCase`

Keep `assignCase`